docs/http-api.md

```diff
@@ -66,6 +66,47 @@ json and treejson are the same.
 curl -H "X-Org-Id: 12345" "http://localhost:6060/metrics/find?query=statsd.fakesite.counters.session_start.*.count"
 ```
 
+## Find tagged metrics
+
+```
+GET /tags/findSeries
+POST /tags/findSeries
+```
+
+Returns metrics which match tag queries and have received an update since `from`.
+Note: the returned results are not deduplicated and in certain cases it is possible
+that duplicate entries will be returned.
+
+##### Parameters
+
+* expr (required): a list of [tag expressions](#tag-expressions)
+* from: Graphite [from time specification](#fromto) (optional. defaults to now-24hours)
+* format: series-json, lastupdate-json. (defaults to series-json)
```

Comment:
opinion:
1) lastUpdate (and lastWrite for that matter) are poorly named variables. ([details](https://github.com/grafana/metrictank/blob/master/docs/metadata.md#lastsave-vs-lastupdate)). This rename would make sense:
  - lastUpdate -> lastTs
  - lastWrite -> lastUpdate
2) "lastUpdate-json" -> not immediately clear what this means exactly.

it seems like you agree with point 1, considering you use "lastTs" in the response data.
I suggest we use format "lastts-json", that way it's consistent with your output and also solves my point 2.
we can use "lastUpdate" internally in the code and internal api's, but everything userfacing can use lastTs/lastts/lastTimestamp.